result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+          type: string
+        chassis_id_type:
+          description: >-
+            This field identifies the format and source of the chassis identifier string. 
+            It is an enumerator defined by the LldpChassisIdSubtype object from IEEE 802.1AB 
+            MIB.
+          type: string
+          enum: [port_component, network_address,chassis_component,mac_address,interface_name,local,interface_alias]
+        id:
+          description: >-
+            System generated identifier for the neighbor on the interface.
+          type: string
+        age:
+          description: >-
+            Age since discovery in seconds.
+          type: integer
+          format: int64
+        last_update:
+          description: >-
+            Seconds since last update received.
+          type: integer
+          format: int64
+        ttl:
+          description: >-
+            The time-to-live (TTL) in seconds is a mandatory TLV which indicates how long information from the neighbor 
+            should be considered valid.
+          type: integer
+        port_id:
+          description: >-
+            The Port ID is a mandatory TLV which identifies the port component of the endpoint identifier associated with 
+            the transmitting LLDP agent. If the specified port is an IEEE 802.3 Repeater port, then this TLV is optional.
+          type: string
+        port_id_type:
+          description: >-
+            This field identifies the format and source of the port identifier string. It is an enumerator defined by the
+            PtopoPortIdType object from RFC2922.
+          type: string
+          enum: [port_component, network_address,agent_circuit_id,mac_address,interface_name,local,interface_alias]
+        port_description:
+          description: >-
+            The binary string containing the actual port identifier for the port which this LLDP PDU was transmitted. The source 
+            and format of this field is defined by PtopoPortId from RFC2922.
+          type: string
+        management_address:
+          description: >-
+            The Management Address is a mandatory TLV which identifies a network address associated with the local LLDP agent, which 
+            can be used to reach the agent on the port identified in the Port ID TLV.
+          type: string
+        management_address_type:
+          description: >-
+            The enumerated value for the network address type identified in this TLV. This enumeration is defined in the 'Assigned Numbers' 
+            RFC [RFC3232] and the ianaAddressFamilyNumbers object.
+          type: string
+        custom_tlvs:
+          type: array
+          items:
+            $ref: '#/components/schemas/LldpCustomTLV.State'
+        capabilities:
+          type: array
+          items:
+            $ref: '#/components/schemas/LldpCapability.State'
+
+    LldpCustomTLV.State:
+      description: >-
+        Custom LLDP TLV received from a neighbor.
```

Comment:
It would be good to indicate when a TLV will get added to this list. Some specific pattern of Type indicating Custom Type or absence of type in some specification ? Basically, when would we expect to see some TLV in this list from list of TLVs sent by LLDP neighbor ?

Reply:
updated